app/validators/accession_date_validator.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class AccessionDateValidator < ::ActiveModel::Validator
+  def validate(record)
+    return if record.date.blank?
+    return if record.date.is_a?(DateTime)
+    record.errors.add(:date, "Dates should be DateTime objects")
```

Comment:
Probably something like "Please enter dates as MM/DD/YYYY". once coersion is working it will accept a variety of inputs, but we probably just need to choose one to recommend. "YYYY/MM/DD" would be another option. I don't know whether figgy has an opinion or recommendation on this anywhere else.

Reply:
Changed the error message to the first suggestion.